utils/index.js

```diff
@@ -26,3 +26,8 @@ exports.formatEventDateTime = (event) => ({
   start_time: formatTime(event.start_time),
   end_time: formatTime(event.end_time),
 });
+
+exports.formatEventTypeDate = (event) => ({
```

Comment:
I think `event` here should be `eventType`.

Reply:
Yeah, good idea!